src/app/component/eco-news/components/create-news/create-news.component.scss

```diff
@@ -58,10 +58,15 @@ form {
   margin-bottom: 1rem;
 }
 
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
 .wrapper {
   display: grid;
-  grid-template-columns: auto 430px;
-  grid-column-gap: 30px;
+  grid-template-columns: auto 456px;
```

Comment:
why did you use 456?

Reply:
according to the mockup
![create-news(456x231)](https://user-images.githubusercontent.com/58560068/97359000-c4c6d400-18a4-11eb-8db2-47a388f76f4b.png)